frontend/src/app/dashboard/page.tsx

```diff
@@ -21,26 +21,56 @@ const CompaniesPieChart = dynamic(
 const Dashboard: React.FC = () => {
   const [loading, setLoading] = useState(true);
   const [totalCO2Score, setTotalCO2Score] = useState<string>("Loading...");
+  const [monthlyCO2Score, setMonthlyCO2Score] = useState<number>(0);
+  const [monthlyCO2ScoreChange, setMonthlyCO2ScoreChange] = useState<number>(0);
+  const [monthlyGreenTransactions, setMonthlyGreenTransactions] = useState<number>(0);
+  const [monthlyGreenTransactionsChange, setMonthlyGreenTransactionsChange] = useState<number>(0);
   const [circleColor, setCircleColor] = useState<string>("#7d91f5");
   const [isHovered, setIsHovered] = useState(false);
   const router = useRouter();
 
   useEffect(() => {
-    const fetchTotalCO2Score = async () => {
+    const fetchDashboardData = async () => {
       try {
-        const response = await fetch("https://rattm-f300025e7172.herokuapp.com/dashboard/get_total_co2_score/");
-        const data = await response.text();
-        setTotalCO2Score(data); 
+        const [
+          totalCO2Response,
+          monthlyCO2Response,
+          co2ChangeResponse,
+          greenTransactionsResponse,
+          greenTransactionsChangeResponse
+        ] = await Promise.all([
+          fetch("https://rattm-f300025e7172.herokuapp.com/dashboard/get_total_co2_score/"),
+          fetch("https://rattm-f300025e7172.herokuapp.com/dashboard/get_this_month_co2_score/"),
+          fetch("https://rattm-f300025e7172.herokuapp.com/dashboard/get_co2_score_change/"),
+          fetch("https://rattm-f300025e7172.herokuapp.com/dashboard/get_this_month_green_transactions/"),
+          fetch("https://rattm-f300025e7172.herokuapp.com/dashboard/get_green_transaction_change/")

```

Comment:
It's not great practice to be re-writing the full Heroku URL so many times (this is not just an issue for this file). We should probably be accessing the root URL from a .env variable. I can make a PR to refactor this across the entire project.